### ISSUE TITLE
!retweet

### ISSUE THREAD

**Issue by jfhbrook** [nodejitsu/kohai#45]
The ability for the twitter plugin to retweet the latest tweet would be pretty rockin'.

**Comment by dominictarr** [comment]
infinite twitter loop!

**Comment by Marak** [comment]
Yeah....except unless we start appending a unique id to each tweet to track internally, this will not really work.

There is a high likelihood that by the time you type `!retweet` and hit enter, that another tweet will come in and you will retweet the wrong tweet!

**Comment by chapel** [comment]
@marak is right, there would have to be an ID or some way to specify what to RT. Maybe do a hash of the tweet, or tweet id, so you could just !retweet <hash> and it would handle the rest.
